.stylelintrc.json

```diff
@@ -1,14 +1,11 @@
 {
-  "plugins": [
-    "stylelint-scss"
-  ],
+  "extends": "stylelint-config-standard-scss",
   "rules": {
       "scss/at-import-no-partial-leading-underscore": true,
       "scss/dollar-variable-colon-space-after": "at-least-one-space",
       "scss/dollar-variable-default": true,
 
       "color-no-invalid-hex": true,
-      "function-calc-no-invalid": true,
```

Comment:
https://stylelint.io/migration-guide/to-14/#function-calc-no-invalid-rule la règle disparait